### PR TITLE
chore: Docs tweaks + clarifications

### PR DESCRIPTION
**Pull request by martijneken**
Clarify REPIN command + ABI.

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -81,30 +81,30 @@ here: https://github.com/proxy-wasm/spec/tree/master
...
     *   on_done
     *   on_delete
-*   Stream context HTTP callbacks
+*   Stream context HTTP callbacks (host -> wasm)
     *   on_request_headers
     *   on_response_headers
-*   Stream context HTTP methods
+*   Stream context HTTP methods (wasm -> host)
     *   send_local_response
     *   get__header_map_value, add_header_map_value, replace_header_map_value,
```
Comment (mpwarres) [inline plugins/README.md:99]: nit: looks like there's an extra '_' in get_header_map_value

File: plugins/README.md [hunk 2]
```diff
@@ -81,30 +81,30 @@ here: https://github.com/proxy-wasm/spec/tree/master
...
+*   Stream context lifecycle callbacks (host -> wasm)
     *   on_context_create
     *   on_done
     *   on_delete
-*   Stream context HTTP callbacks
+*   Stream context HTTP callbacks (host -> wasm)
     *   on_request_headers
     *   on_response_headers
-*   Stream context HTTP methods
+*   Stream context HTTP methods (wasm -> host)
```
Comment (mpwarres) [inline plugins/README.md:97]: optional: s/methods/hostcalls/ ? That's the term used in both ABI and C++ SDK specs for wasm -> host invocations.

File: plugins/README.md [hunk 3]
```diff
@@ -81,30 +81,30 @@ here: https://github.com/proxy-wasm/spec/tree/master
...
     *   send_local_response
     *   get__header_map_value, add_header_map_value, replace_header_map_value,
         remove_header_map_value
     *   get_header_map_pairs, set_header_map_pairs
     *   get_header_map_size
-*   Other methods
+*   Other methods (wasm -> host)
```
Comment (mpwarres) [inline plugins/README.md:103]: s/methods/hostcalls/? (see prev comment)

**Review by mpwarres — APPROVED** [review]

File: plugins/README.md (hunk 1 above)
Comment (martijneken) [reply plugins/README.md]: Done

File: plugins/README.md (hunk 2 above)
Comment (martijneken) [reply plugins/README.md]: Done

File: plugins/README.md (hunk 3 above)
Comment (martijneken) [reply plugins/README.md]: Done

**Pull request merged**